src/test/java/uk/gov/hmcts/darts/audio/service/impl/AudioOperationServiceImplTest.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.darts.audio.service.impl;
+
+import org.apache.commons.exec.CommandLine;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.darts.audio.model.AudioFileInfo;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SpringBootTest
+class AudioOperationServiceImplTest {
```

Comment:
I think this should probably live in `src/integrationTest`?

Reply:
It should change to Mockito extension as its intended for unit test case